egg.js

```diff
@@ -3,13 +3,30 @@ require("dotenv"); // Load env configuration
 const Discord = require('discord.js');
 const client = new Discord.Client();
 
+let eggCount = 0;
+
 client.on('ready', () => {
   console.log(`Logged in as ${client.user.tag}!`);
+  client.user.setPresence({
+    game: { 
+	  name: `${eggCount} eggs.`,
+	  type: 'WATCHING'
+	},
+	status: 'idle'
+  })
 });
 
 client.on('message', msg => {
   if(msg.content.includes("egg")){
-    msg.react('🥚');
+	msg.react('🥚');
+	eggCount++;
+	client.user.setPresence({
+      game: { 
+        name: `${eggCount} eggs.`,
```

Comment:
How about `${eggCount} 🥚` instead? The bot doesn't really talk, so an emoji seems more appropriate.

Reply:
That sounds like a good idea.